fdbserver/DDRelocationQueue.actor.cpp

```diff
@@ -1875,8 +1887,9 @@ ACTOR Future<Void> dataDistributionRelocator(DDQueue* self,
 						dataTransferComplete.send(rd);
 					}
 
+					int nonOverlappingCount = nonOverlappedSrcCount(rd.completeSources, destIds);
```

Comment:
Nit: This can be `const`. (Or, you could use the function directly in line 1892 below.)

Reply:
I added `const` here